django_countries/filters.py

```diff
@@ -22,8 +23,12 @@ def choices(self, changelist):
             "display": _("All"),
         }
         for lookup, title in self.lookup_choices(changelist):
+            if django.VERSION >= (5, 0):
+                selected = force_str(lookup) in value
```

Comment:
I had to change this line so it works when no country is selected in the CountryFilter `list_filter` I'm using, otherwise it'll throw a TypeError exception that `argument of type 'NoneType' is not iterable`

```suggestion
                selected = value and force_str(lookup) in value
```

Reply:
@adamchainz any onjection to this?